src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.model;
+
+import java.util.Arrays;
+
+public class FruitTransaction {
+    public enum Operation {
```

Comment:
The `Operation` enum is public. Consider limiting its visibility if it's only used within the `FruitTransaction` class. This would follow the encapsulation principle.